docs/user/security/reporting.asciidoc

```diff
@@ -9,19 +9,56 @@ To use {reporting} with {security} enabled, you need to
 <<using-kibana-with-security,set up {kib} to work with {security}>>.
 If you are automatically generating reports with
 {ref}/xpack-alerting.html[{watcher}], you also need to configure {watcher}
-to trust the {kib} server's certificate. For more information, see
+to trust the {kib} server's certificate. 
+////
+For more information, see
 <<securing-reporting>>.
+////
 
 [[reporting-app-users]]
-To enable users to generate reports, assign them the built-in `reporting_user`
+To enable users to generate reports, you must assign them the built-in `reporting_user`
 role.  Users will also need the appropriate <<kibana-privileges, {kib} privileges>> to access the objects 
 to report on and the {es} indices.
 
-* If you're using the `native` realm, you can assign roles through
-**Management > Users** UI in Kibana or with the `user` API. For example,
-the following request creates a `reporter` user that has the
-`reporting_user` role and the `kibana_user` role:
+
+==== If you are using the `native` realm
+
+You can assign roles through the
+*Management* app in Kibana or with the {ref}/security-api-put-user.html[user API]. 
```

Comment:
The `user API` phrase links away from the page, but it really seems relevant to a block of text below on the same page, that starts with `This example uses the user API to create a user...`